src/frontend/src/eth/derived/erc20.derived.ts

```diff
@@ -116,7 +117,7 @@ export const enabledErc20TokensAddresses: Readable<ContractAddressText[]> = deri
 
 export const erc20UserTokensInitialized: Readable<boolean> = derived(
 	[erc20UserTokensStore],
-	([$erc20UserTokensStore]) => $erc20UserTokensStore !== undefined
+	([$erc20UserTokensStore]) => nonNullish($erc20UserTokensStore)
```

Comment:
Are you sure this was not checked for `undefined` on purpose given that `null` in store is often used to express the fact that something was explicitely set to `null`?